workers/Dockerfile.tests

```diff
@@ -0,0 +1,3 @@
+FROM dr_worker
```

Comment:
Hmm. GitHub is being slow with your comments so mine is out of order, but I follow you.